### PR TITLE
Links to documentation + community in the readme. Add contributing.md

### PR DESCRIPTION
**Pull request by louim**
Hopefully this will help reduce the amount of support issues open by channeling people towards the correct ressources. :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,3 +143,20 @@ Full SSL support is available for your WordPress sites. Trellis will also *auto-
 ## Caching
 
 You can enable FastCGI caching on a per site basis. The cache is a short duration, "micro-cache" type setup. See the [FastCGI micro-caching docs](https://roots.io/trellis/docs/fastcgi-caching/) for configuration options.
+
+## Documentation
```
Comment (swalkinshaw) [inline README.md]: Can we actually move this further up? Maybe just before Requirements?

Comment (louim) [inline README.md]: Sure thing!

**Comment by louim** [comment]
@swalkinshaw I forgot you don't get a notification when something is pushed. :sweat_smile: I moved the documentation up. Let me know if this is good to go!

**Comment by swalkinshaw** [comment]
Thanks!

**Pull request merged**